atox/src/main/kotlin/MainActivity.kt

```diff
@@ -1,4 +1,4 @@
-// SPDX-FileCopyrightText: 2019-2022 aTox contributors
+// SPDX-FileCopyrightText: 2019-2021 aTox contributors
```

Comment:
Why?

Reply:
This is just 3 `git revert`s with no changes other than resolving conflicts with other changes made since. :P We'll update the copyright again next time we touch the file.